compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -2101,6 +2101,7 @@ fn add_apple_sdk(cmd: &mut dyn Linker, sess: &Session, flavor: LinkerFlavor) {
         ("aarch64", "tvos") => "appletvos",
         ("x86_64", "tvos") => "appletvsimulator",
         ("arm", "ios") => "iphoneos",
+        ("aarch64", "ios") if llvm_target.contains("macabi") => "macos11",
```

Comment:
This doesn't have `x` like the similar target below. How can we double check this value?

Reply:
You were right, it was the wrong value. I've updated it accordingly.